opengemini/command_test.go

```diff
@@ -192,20 +192,12 @@ func TestClient_ShowTagValues(t *testing.T) {
 		With("location").Limit(2).Offset(0))
 	assert.Nil(t, err)
 	assert.Equal(t, 2, len(tagValueResult))
-	expValues = []string{"c1", "c2"}
```

Comment:
why delete these test cases?

Reply:
When the server executes ·show tag value·, the returned results become out of order. The server is required to check for recent changes.